src/test/java/com/example/tournamentMaker/tournament/result/ResultServiceTest.java

```diff
@@ -0,0 +1,90 @@
+package com.example.tournamentMaker.tournament.result;
+
+import com.example.tournamentMaker.statistics.FootballStatistics;
+import com.example.tournamentMaker.team.Team;
+import com.example.tournamentMaker.team.player.FootballPlayer;
+import com.example.tournamentMaker.team.player.PlayerRepository;
+import com.example.tournamentMaker.util.Util;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class ResultServiceTest {
+    @InjectMocks
+    private ResultService resultService;
+    @Mock
+    private PlayerRepository playerRepository;
+    private final Util util = new Util();
+
+    @Test
+    void shouldUpdateGoalsCount() {
+        //given
+        int hostPoints = 3;
+        int guestPoints = 2;
+        FootballStatistics hostStatistics = new FootballStatistics(util.createTeam("Team A"));
+        FootballStatistics guestStatistics = new FootballStatistics(util.createTeam("Team B"));
+        //when
+        resultService.updateGoalsCount(hostPoints, guestPoints, hostStatistics, guestStatistics);
+        //then
+        Assertions.assertAll(
+                () -> assertEquals(hostPoints, hostStatistics.getGoalsScored()),
+                () -> assertEquals(hostPoints, guestStatistics.getGoalsConceded()),
+                () -> assertEquals(guestPoints, guestStatistics.getGoalsScored()),
+                () -> assertEquals(guestPoints, hostStatistics.getGoalsConceded())
+        );
+    }
+
+    @Test
+    void shouldUpdateSpecificStatisticInTeam() {
+        //given
+        List<Integer> jerseyNumbersList = List.of(5, 5, 7);
+        Team team = util.createTeam("Team A");
+        FootballPlayer player1 = util.createFootballPlayer(1L, "Jack", team, 5);
+        FootballPlayer player2 = util.createFootballPlayer(3L, "Will", team, 7);
+        Map<Long, Integer> specificStatistic = new HashMap<>();
+        specificStatistic.put(1L, 1);
+        specificStatistic.put(3L, 22);
+        //when
+        when(playerRepository.findByJerseyNumberAndTeam(5, team)).thenReturn(Optional.of(player1));
+        when(playerRepository.findByJerseyNumberAndTeam(7, team)).thenReturn(Optional.of(player2));
+        resultService.updateSpecificStatisticInTeam(jerseyNumbersList, team, specificStatistic);
+        //then
+        assertEquals(3, specificStatistic.get(player1.getId()));
+        assertEquals(23, specificStatistic.get(player2.getId()));
+    }
+
+    @Test
+    void shouldUpdateSpecificStatisticInTeamWhenNoPlayersInSpecificStatisticMap() {
+        //given
+        List<Integer> jerseyNumbersList = List.of(5, 5, 5, 5);
```

Comment:
czego tu sa same 5? bo to sa numery koszulek to co ? 4x jest chlop z numerem 5 w jednej druzynie ? 
Nie ma to dla mnie sensu, ale moze nie rozumiem kontektstu

Reply:
Gość z numerem 5 strzelił 4 bramki

Reply:
troche glipio to wyglada, nie lepiej by bylo miec 5,4 ? 
Albo miec zmienna ktora trzyma ilosc bramek po prostu ? 
Bo tka to wychodzi na to ze do listy bedziesz pchal cztery razy typa z numerem 5. 
Nie podoba mi sie to